### PR TITLE
Fix exit dialog bugs and automatically change trim mode to polygon when polygon algorithm is selected

### PR DESCRIPTION
**Pull request by TheCodez**
The implementation should now be complete and every property change should get registered

### CODE REVIEW

File: SpriteSheetPacker/MainWindow.cpp [hunk 1]
```diff
@@ -779,16 +784,16 @@ void MainWindow::removeScalingVariant() {
     qDebug() << "remove" << sender();
     ScalingVariantWidget* scalingVariantWidget = qobject_cast<ScalingVariantWidget*>(sender());
     if (scalingVariantWidget) {
+        disconnect(scalingVariantWidget, SIGNAL(valueChanged()), this, SLOT(scalingVariantWidgetValueChanged()));
```
Comment (amakaseev) [inline SpriteSheetPacker/MainWindow.cpp]: delete widget don remove slot connection automate?

Comment (TheCodez) [inline SpriteSheetPacker/MainWindow.cpp]: Don't know just for safety.

**Comment by amakaseev** [comment]
I check everything tomorrow, thank you very much for the work done :+1:

**Comment by TheCodez** [comment]
No problem haven't finished the progress bar change though.

**Pull request merged**